flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaTableTestBase.java

```diff
@@ -46,7 +46,7 @@
 @RunWith(Parameterized.class)
 public abstract class KafkaTableTestBase extends KafkaTestBaseWithFlink {
 
-	private static final String JSON_FORMAT = "json";
+	protected static final String JSON_FORMAT = "json";
```

Comment:
How about add format: 
```	
protected static final String DEBEZIUM_JSON = "debezium-json";
....
//add parameters
new Object[]{false, DEBEZIUM_JSON}
...
	@Test
	public void testKafkaSourceSink() throws Exception {
		if (DEBEZIUM_JSON.equals(format)) {
			// skip debezium json format
			return;
		}
       ....
```
And then the testKafkaDebeziumChangelogSource will be more clear like:
```
if (isLegacyConnector || !DEBEZIUM_JSON.equals(format)) {
			// skip tests for legacy connector, because changelog source is only supported in new connector
			return;
		}
```